flash/core/trainer.py

```diff
@@ -239,3 +242,61 @@ def from_argparse_args(cls, args: Union[Namespace, ArgumentParser], **kwargs) ->
         # the lightning trainer implementation does not support subclasses.
         # context: https://github.com/PyTorchLightning/lightning-flash/issues/342#issuecomment-848892447
         return from_argparse_args(Trainer, args, **kwargs)
+
+    @property
+    def estimated_stepping_batches(self) -> Union[int, float]:
```

Comment:
quick ques: why did you reimplemented it here instead of taking it from PL?

Reply:
To support PL versions pre 1.6 when it was added, anything 1.6 or higher it just calls to the super class